lib/wanda/executions/target.ex

```diff
@@ -19,4 +21,10 @@ defmodule Wanda.Executions.Target do
     |> Enum.flat_map(& &1.checks)
     |> Enum.uniq()
   end
+
+  def from_list(map_list) do
```

Comment:
this needs to be tested

Reply:
tested here https://github.com/trento-project/wanda/pull/73/commits/694288d46fa08ac3a6ba43e1016043cdfa22c7d3